core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -24,9 +27,11 @@
     private final EncodingManager encodingManager;
     private final int motorVehicleInt;
 
+    private final double DELTA = 0.1;
+
     public DataFlagEncoderTest() {
         encoder = new DataFlagEncoder();
-        encodingManager = new EncodingManager(encoder);
+        encodingManager = new EncodingManager(Arrays.asList(encoder), 8);
```

Comment:
Sorry guys, didn't get the point here. What do you suggest in terms of configurable number of encoders? I understand the current limit of 32 bit or 64 bit for flags.